tests/php/Service/ExAppSetupCheckServiceTest.php

```
@@ -0,0 +1,99 @@
+			return true;
+		});
+		$this->service->optOut('myapp');
+		self::assertArrayNotHasKey('myapp', $stored['apps']);
+		self::assertArrayHasKey('other', $stored['apps']);
+	}
+
+	public function testGetOptedInAppIdsFiltersPrefixAndExtractsAppId(): void {
+		$this->appConfig->method('getKeys')->with('app_api')
+			->willReturn(['version', 'loglevel', 'setup_checks_appA', 'setup_checks_live_transcription', 'setupchecks_state']);
+		// state key ('setupchecks_state') deliberately does NOT match the prefix and must be excluded.
+		self::assertSame(['appA', 'live_transcription'], $this->service->getOptedInAppIds());
```

PHPStan (2.2.2)

[error] Call to an undefined static method OCA\AppAPI\Tests\php\Service\ExAppSetupCheckServiceTest::assertSame().


(staticMethod.notFound)